plugins/SitesManager/lang/en.json

```diff
@@ -147,15 +147,15 @@
         "CloudflareDetectedEmail": "Cloudflare was detected on your website. Did you know you can use the Matomo app in your Cloudflare setup to track the website data? Learn more here: %1$s",
         "SiteWithoutDataGoogleTagManagerIntro": "If you use Google Tag Manager, you can start tracking data in Matomo seamlessly using the \"Custom HTML Tag\" available in Google Tag Manager. %1$sLearn more%2$s.",
         "SiteWithoutDataGoogleTagManagerFollowStep1": "%1$sLogin%2$s into your Google Tag Manager account.",
-        "SiteWithoutDataGoogleTagManagerFollowStep2": "Click \"Tags\" and then \"New\".",
-        "SiteWithoutDataGoogleTagManagerFollowStep3": "Click \"Tag Configuration\" and select \"Custom HTML\"",
-        "SiteWithoutDataGoogleTagManagerFollowStep4": "Copy the standard Matomo JavaScript Tracking code as provided below.",
-        "SiteWithoutDataGoogleTagManagerFollowStep5": "Paste this JavaScript tracking code in the Tag Configuration HTML field.",
-        "SiteWithoutDataGoogleTagManagerFollowStep6": "Scroll down and click on \"Triggering\".",
-        "SiteWithoutDataGoogleTagManagerFollowStep7": "Select \"All Pages\" to fire the \"Matomo tracking tag\" on all of your pages.",
-        "SiteWithoutDataGoogleTagManagerFollowStep8": "Click \"Save\".",
-        "SiteWithoutDataGoogleTagManagerFollowStep9": "Preview your changes by clicking the \"Preview\" button.",
-        "SiteWithoutDataGoogleTagManagerFollowStep10": "Check if the Matomo tag is firing as expected.",
+        "SiteWithoutDataGoogleTagManagerFollowStep2": "Select the container that you want to add Matomo to.",
+        "SiteWithoutDataGoogleTagManagerFollowStep3": "Click \"Tags\" and then \"New\".",
+        "SiteWithoutDataGoogleTagManagerFollowStep4": "At the top of the window, provide a name for your tag.",
+        "SiteWithoutDataGoogleTagManagerFollowStep5": "Click \"Tag Configuration\" and select \"Custom HTML\".",
+        "SiteWithoutDataGoogleTagManagerFollowStep6": "Copy the Matomo JavaScript Tracking code as provided below.",
+        "SiteWithoutDataGoogleTagManagerFollowStep7": "Paste this JavaScript tracking code in the Tag Configuration HTML field.",
+        "SiteWithoutDataGoogleTagManagerFollowStep8": "Scroll down and click on \"Triggering\".",
+        "SiteWithoutDataGoogleTagManagerFollowStep9": "Select \"All Pages\" to fire the \"Matomo tracking tag\" on all of your pages.",
+        "SiteWithoutDataGoogleTagManagerFollowStep10": "Click \"Save\".",
```

Comment:
@michalkleiner As basically every translation changed its content it would indeed be better to use new keys. This might not hurt on 4.x-dev, but as we plan to do that on 5.x-dev as well it might be better to change it on both branches.